completion/git-machete.completion.zsh

```diff
@@ -34,6 +34,12 @@ _git-machete() {
             '(-H --sync-github-prs)'{-H,--sync-github-prs}'[Annotate with GitHub PR numbers and authors where applicable]' \
           && ret=0
           ;;
+        (clean)
```

Comment:
Actually! I'd add some extra output to `clean` to make it clear what happens in the subsequent stages. Consider the following output:

```
➜ 14:35 ~/git-machete feature/command_clean PR #435 (amalota) $ git machete clean 
No branches to delete
Delete branch bugfix/circleci-deploy-to-run (merged to HEAD)? (y, N, q) q
➜ 14:36 ~/git-machete feature/command_clean PR #435 (amalota) $ 
```

It's pretty weird to see `No branches to delete`... only to get a prompt about deleting a branch 😅

So `Checking for untracked managed branches with no downsteam...` and `Checking for unmanaged branches...`, probably in bold, would be pretty useful